flybywiresim/aircraft
# feat(elec): add A380X electrical failures to EFB failure page

Fixes #[issue_no]

## Summary of Changes

Adds electrical failures for the A380X to the EFB.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions



## How to download the PR for QA
Test all the electrical failures and see if the corresponding component is failed (by using SD elec pages or the behavior menu).

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
